network/errors.go

```diff
@@ -0,0 +1,33 @@
+package network
+
+import "fmt"
+
+// NotSubscribedError is returned when the peer is not subscribed to a
+// specific topic.
+type NotSubscribedError struct {
+	TopicID TopicID
+}
+
+func (e NotSubscribedError) Error() string {
+	return fmt.Sprintf("Not subscribed to the '%s' topic", e.TopicID.String())
```

Comment:
it would be better to whole of error message be lowercase (based on IDE linter warnings)

Reply:
Thanks. Please check and resolve it